From f24498950d44235e307e6734ca627723e34986a9 Mon Sep 17 00:00:00 2001
From: "alexey.lysiuk" <alexey.lysiuk@gmail.com>
Date: Sun, 14 Mar 2021 19:36:16 +0200
Subject: [PATCH] - made sndfile imported library global

CMake earlier than 3.18 does not support library aliases to non-global imported targets.
This fixes #22.
---
 cmake/FindSndFile.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git cmake/FindSndFile.cmake cmake/FindSndFile.cmake
index 0266980..7ba3225 100644
--- cmake/FindSndFile.cmake
+++ cmake/FindSndFile.cmake
@@ -27,7 +27,7 @@ include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(SndFile DEFAULT_MSG SNDFILE_LIBRARY SNDFILE_INCLUDE_DIR)
 
 if(SNDFILE_FOUND)
-	add_library(sndfile UNKNOWN IMPORTED)
+	add_library(sndfile UNKNOWN IMPORTED GLOBAL)
 	set_target_properties(sndfile
 	PROPERTIES
 		IMPORTED_LOCATION "${SNDFILE_LIBRARY}"
-- 
2.31.1

